Gemfile.lock

```diff
@@ -361,6 +390,7 @@ DEPENDENCIES
   rails!
   rake (>= 10.3)
   redcarpet (~> 3.2.3)
+  redis (~> 3.0)
```

Comment:
@dhh I think this would work.  Will this have any after effects when someone is trying to upgrade from 4.2.x to 5?